### ISSUE TITLE
Passphrase wildcards escaping braces characters

### ISSUE THREAD

**Issue by s0rl0v** [fixmycrypto/FixMyCrypto#2]
Hello,

Thank you for making such useful project.

https://github.com/fixmycrypto/FixMyCrypto?tab=readme-ov-file#passphrase-wildcards

For example, I want to use this passphrase wildcard:
```
"(_l)uck_||_f)st_)[0-9]<4>"
```

However the app enumerates it as:
```
"_luck_||_f)st_)0000"
```

This could be seen in checkpoin.json if one to specify some complex enough passphrase to make the system to make checkpoin files.

Could you advise how to escape braces in boolean expressions? Or how I can make use of boolean expressions in passphrases with braces?

Thank you and have a great day.

**Comment by fixmycrypto** [comment]
Hi, you should be able to use square brackets to escape one parenthesis e.g.:
`(   [)]    )`

Or parenthesis to escape a square bracket:
`[    (])    ]`

Make sure you have the latest version because I fixed a bug related to this recently. If this doesn't work let me know.

**Comment by s0rl0v** [comment]
Yeah, I tried escaping it like you mentioned - it still treats ) like termination of the expression.
I use the latest version btw.
